frontend/scripts/makefile/flutter.toml

```diff
@@ -173,12 +217,31 @@ script = ["""
   """]
 script_runner = "@shell"
 
-[tasks.flutter-build.windows]
+[tasks.flutter-build.linux]
```

Comment:
Why do you delete the `[tasks.flutter-build.windows]`?

Reply:
Oh didn't know had deleted it. Must have happened when I was trying to solve merge conflicts, will fix it.